matscipy/fracture_mechanics/crack.py

```diff
@@ -54,6 +62,208 @@
 MPa_sqrt_m = 1e6*units.Pascal*np.sqrt(units.m)
 
 ###
+# Temporary function. Eventually meant to be pushed to ase.optimize.ode
```

Comment:
Since this will not happen fast maybe we should move it to a new `matscipy.optimize` module?